content/sensu-enterprise/3.3/guides/email-gsuite.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Using the Email Integration with G Suite"
+linkTitle: "Email with Gsuite"
+product: "Sensu Enterprise"
+version: "3.3"
+weight: 1
+menu:
+ sensu-enterprise-3.3:
+   parent: guides
+---
+
+In this guide, we'll walk through how to configure the Sensu Enterprise email integration to use G Suite as the SMTP gateway for sending email alerts.
+
+Before we configure Sensu to use the gateway, we'll have to first set up a user and app password in G Suite to use in our email integration configuration.
+
+## Set Up User and App Password
+
+The first step to using G Suite as our SMTP relay for Sensu is to set up a user and an app password. We generally recommend adding Sensu user to be used specifically with our deployment. If this isn't possible, we can use any G Suite user account.
+
+Once we have an account that will be used for sending email via Sensu, we'll then need to [create an app password][1]. When we generate the password, we need to ensure that we label it with "Sensu" so that we'll know what the app password is being used for.
+
+After we have our user and app password created, we can then move on to configuring the integration.
+
+## Email Integration Configuration
+
+Now that we have the credentials we need to send email via G Suite, we'll need to review [G Suite's SMTP settings][2].
+
+In the case of this guide, we'll use the following as our settings:
+
+SMTP address: smtp-relay.gmail.com
+Port: 587
+
+Let's take a look at what the full configuration looks like:
+
+{{< highlight json >}}
```

Comment:
```suggestion
{{< highlight test >}}
```

Reply:
This will resolve the failure and prevent any red highlighting. Or you could separate the filename from the JSON and keep the JSON highlighting

Reply:
IMO, the file name should be removed from the JSON block. The point of that JSON check is to provide a checker for valid JSON so users can copy/paste things properly. In having at least another case where we are changing the tags to make a test pass, it's further going away from that idea. We already have cases where multiple JSON blocks fail and require a `shell` tag instead to get around. Keeping those to a minimum would be ideal I think, although I'd like to revisit this eventually in another language to see if other JSON libraries are more relaxed.

Also, if we ever get the copy button working for code blocks, the file name makes that copy less useful as the user will have to remove the file name from their paste.

Not sure if markdown has it (quick search and I didn't find it), but in Jira you could name your code blocks with a title and it would appear at the top of the code block. Maybe there is some formatting we could apply to the file name to fake it above the block.

Reply:
I still can't find a way to name a code block. I see two options immediately:

- Bold the file name above the code block using `**`:

**/etc/sensu/conf.d/handlers/email.json**

- Call out the file name in the sentence above the code block:

```
SMTP address: smtp-relay.gmail.com
Port: 587

Let's take a look at what the full configuration looks like using `/etc/sensu/conf.d/handlers/email.json`:

{{< highlight json >}}

{
```